v2/client.go

```diff
@@ -425,6 +426,13 @@ type Client struct {
 	Logger     *log.Logger
 	TimeOffset int64
 	do         doFunc
+
+	UsedWeight UsedWeight
+}
+
+type UsedWeight struct {
+	Used   int64
+	Used1M int64 // used in last 1 minute
```

Comment:
since we're adding this, I think we can also add the order-count

```
'X-Amz-Cf-Id': 'RGIpL0I5fdkln_UMPINwbS-1wpyOn-MhCwZmK8CiLuzcQSdYmd7OsQ==',
  'X-Amz-Cf-Pop': 'MAD56-P3',
  'X-Cache': 'Miss from cloudfront',
  'X-Content-Security-Policy': "default-src 'self'",
  'X-Content-Type-Options': 'nosniff',
  'X-Frame-Options': 'SAMEORIGIN',
  'X-Mbx-Order-Count-10s': '1',
  'X-Mbx-Order-Count-1d': '1',
  'X-Mbx-Used-Weight': '21',
  'X-Mbx-Used-Weight-1m': '21',
  'X-Mbx-Uuid': 'fa1a9f25-7d4b-4fc9-ba80-c402e0ddef36',
  'X-Webkit-Csp': "default-src 'self'",
  'X-Xss-Protection': '1; mode=block'
} 
```